### PR TITLE
Equipment and mechanics

### PR DESCRIPTION
**Pull request by zbeermann**
Added mechanic instance gets, equipment category and subcategory post/get

### CODE REVIEW

File: pydnd/equipment/views.py [hunk 1]
```diff
@@ -1,18 +1,75 @@
...
+    queryset = EquipmentCategory.objects.all()
+    serializer_class =  EquipmentCategorySerializer
+
+
+class EquipmentCategoryGet(APIView):
+
+    serializer_class = EquipmentCategorySerializer
+
+    def get(self, request, name_or_id):
+        return get_equipment_category(name_or_id)
```
Comment (pyasi) [inline pydnd/equipment/views.py]: This is probably specific enough to not be it's own method and just live here in this GET.

File: pydnd/mechanics/views.py [hunk 2]
```diff
@@ -24,3 +25,73 @@ class DamageTypeList(generics.ListCreateAPIView):
 class MagicSchoolList(generics.ListCreateAPIView):
     queryset = MagicSchool.objects.all()
     serializer_class = MagicSchoolListSerializer
+
+class LanguageGet(APIView):
+
+    serializer_class = LanguageListSerializer
+
+    def get(self, request, name_or_id):
+        return get_language(name_or_id)
```
Comment (pyasi) [inline pydnd/mechanics/views.py]: I would say ditto to all of these as the one above.

UNLESS we wanted to make some super generic class like `get_model(model_name, name_or_id` and have `model_name.objects.get()` in each. That could work.

File: scripts/equipmentcategory_script.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+url = "http://127.0.0.1:8000/"
+endpoint = 'equipment/'
+endpoint2 = 'equipment_category/'
+
+response = requests.get(dndurl + endpoint)
+body = response.json()
+count = body['count']
+
+
+for i in range(51, count):
```
Comment (pyasi) [inline scripts/equipmentcategory_script.py:14]: Why  is it starting at 51?

Comment (zbeermann) [reply scripts/equipmentcategory_script.py:14]: 50 and below are armor/weapons

File: pydnd/equipment/views.py [hunk 4]
```diff
@@ -25,3 +82,15 @@ def get(self, request):
         weapon = Weapon.objects.all()
         data = WeaponListSerializer(weapon, many=True).data
         return Response(data)
+
+
+
+def create_attribute(data, attribute_name, serializer, model_type):
+    monster_attributes = []
```
Comment (pyasi) [inline pydnd/equipment/views.py]: Copy paste issue

**Pull request merged**